lib/diagnostic.rb

```diff
@@ -29,31 +31,34 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = "{'#character'} is really a Sith Lord."
 
 # Numbers in Ruby
 #
 # What is the type of object in Ruby for decimal numbers? Write your answer in
 # a Ruby string.
 
-Diagnostic.decimal = 'your response here'
+Diagnostic.decimal = '%'
+#not sure on this.
 
 # What is the type of object in Ruby for integer numbers? Write your answer in
 # a Ruby string.
 
-Diagnostic.integer = 'your response here'
+Diagnostic.integer = '.div or /'
+# not sure with this
 
 # Write an example of a decimal and an integer in Ruby. Replace the Ruby strings
 # in the following array with your examples.
 
-Diagnostic.numbers = ['decimal', 'integer']
+Diagnostic.numbers = ['5 % 2', '5 / 2']
```

Comment:
`5 % 2` evaluates to `1`, which is an integer.  `5 / 2` also evaluates to `1` since neither number is being converted into a floating point number and integer division is performed instead.